### ISSUE TITLE
Breaks text selection everywhere

### ISSUE THREAD

**Issue by xire28** [amcharts/amcharts4#23]
```
<body style="touch-action: none; user-select: none; -webkit-user-drag: none;">
```
`Interaction.prototype.setTemporaryStyle` does not add temporary styles, the DOM 0 styles added to the `<body>` are permanent.
`Interaction.prototype.processTouchable` adds a prevent default listener on `document` for the `pointerdown` event.

Both of them breaks text selection on a application level, this is a major issue.

**Comment by arm1n** [comment]
I guess this is related to #18 due to failed cleanup.

**Comment by xire28** [comment]
No and yes. The fixing the failed clean up will restore text selection feature on the page where no amchart is used but the main issue is related to how amcharts handle touch events.

**Comment by martynasma** [comment]
Technically, those styles should be added to body only when actual dragging is going on, and removed immediately afterwards.

We'll investigate if and why that's not happening.

Thanks!

**Comment by martynasma** [comment]
Fixed in [4.0.0-beta.24](https://github.com/amcharts/amcharts4/releases/tag/4.0.0-beta.24)

[CHANGELOG](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md#400-beta24---2018-05-16)
